sdk/core/azure-mgmt-core/CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
 
 # Release History
 
-## 1.2.1 (unreleased)
+## 1.2.1 (Unreleased)
```

Comment:
Please update this to 2020-10-05 as well. :)

Reply:
i decided not to update mgmt core since @changlong-liu and @RodgeFu own it, and didn't want to force them to release on monday. that being said, @changlong-liu @RodgeFu are you ok with me marking the mgmt core readme with 2020-10-05 release date?

Reply:
They are in a 8-day holiday. I will take care of it.

Reply:
ooh good point, sounds good, i'll udpate the release date